docs/ramalama-serve.1.md

```diff
@@ -273,6 +276,17 @@ $ cat tiny.image
 Image=quay.io/rhatdan/tiny:latest
 ```
 
+### Generate quadlet service off of tiny OCI Model and output to directory
+```
+$ ramalama --runtime=vllm serve --name tiny --generate=quadlet:/etc/containers/systemd oci://quay.io/rhatdan/tiny:latest
```

Comment:
This is misleading, should use sudo, otherwise it looks like a rootless process can write to /etc. Alternately you could write to some where in the users homedir.
~/.config/containers/systemd/